neon_llm_core/chatbot.py

```diff
@@ -24,10 +24,14 @@
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from typing import List
+from typing import List, Optional
```

Comment:
we may omit using `Optional` since Python 3.9 we can replace it with `{type} | None`

Reply:
I see that as [new for 3.10](https://peps.python.org/pep-0604/), so I think we still need to use `Union`/`Optional` for 3.9